### DISCUSSION TITLE
Entendendo metodos.

### DISCUSSION THREAD

**Discussion by cmsouzaac** [balta-io/2801#106] [Perguntas e Respostas] [answered]
Professor na aula sobre metodos, na documentação eu vi que podemos ter dois metodos com os nomes iguais, porem o que diferencia sao seus paramentros? eu nao entendi essa colocação.

![image](https://user-images.githubusercontent.com/48097255/187825686-54fc87ab-d562-47a6-b15a-6a805dc72773.png)

 [https://docs.microsoft.com/pt-br/dotnet/csharp/methods?source=docs](url)

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia @cmsoouza, como vai?

Sim, podemos ter métodos com nomes iguais. O que vale é a assinatura do método, que é composta pelo seu nome e parâmetros.

Desta forma, não tem problema ter o mesmo nome, se houverem parâmetros diferentes. Isto também é conhecido como sobrecarga de método (Method Overload).

**Comment by cmsouzaac** [comment]
blz professor!

**Comment by gilivasconcelos** [comment]
Olá, uma sugestão seria fazer também um curso de orientação a Objetos, segue o do Balta.oi https://balta.io/cursos/fundamentos-orientacao-objetos;

Outra opção é pelo site W3School, ele está em inglês mas você pode usar o Tradutor do Google que ajudará: https://www.w3schools.com/cs/cs_method_overloading.php

**Reply by cmsouzaac** [reply]
opa, pode deixar como eu to fazendo a carreira de backend, ja estou terminando esse modulo de fundamentos e seguirei.
